http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/VPCHelper.scala

```diff
@@ -55,13 +55,20 @@ class VPCHelper(config: VPCHelperConfig,
       )
       .build
 
-  private def networkUri(googleProject: GoogleProject, vpcConfig: VPCConfig): String =
-    s"projects/${googleProject.value}/global/networks/${vpcConfig.value}"
+  def buildNetworkUri(googleProject: GoogleProject, vpcConfig: VPCConfig): String =
```

Comment:
The crux of this fix -- build the right uri for the network or subnet.

This wasn't an issue in the old GCE library -- but it seems like `com.google.cloud.compute.v1` is more strict,